radio/src/sdcard.cpp

```diff
@@ -141,3 +141,47 @@ bool listSdFiles(const char *path, const char *extension, const uint8_t maxlen,
 
   return s_menu_count;
 }
+
+const char *fileCopy(const char *filename, const char *srcDir, const char *destDir)
+{
+  FIL srcFile;
+  FIL dstFile;
+  char buf[256];
+  UINT read = sizeof(buf);
+  UINT written = sizeof(buf);
+
+  char path[2*CLIPBOARD_PATH_LEN+1];
+  char *tmp = strAppend(path, srcDir, CLIPBOARD_PATH_LEN);
+  *tmp++ = '/';
+  strAppend(tmp, filename, CLIPBOARD_PATH_LEN);
```

Comment:
I don't get what you write about CLIPBOARD_PATH_LEN. The directory max len is CLIPBOARD_PATH_LEN, the same for for the file. Therefore the buffer is 2*CLIPBOARD_PATH_LEN and it remains at least CLIPBOARD_PATH_LEN bytes available in path (at position tmp), did I miss anything?